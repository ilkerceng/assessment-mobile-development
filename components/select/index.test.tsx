import React from "react";
import renderer from "react-test-renderer";
import Select from "../select";

const selectProps = {
  type: "multi",
  multiple: true,
  caption: "Filtre",
  items: [
    {
      id: 1,
      name: "Havuz",
    },
    {
      id: 2,
      name: "Spor Salonu",
    },
  ],
};

describe("<Select />", () => {
  test("match snapshot", () => {
    const tree = renderer.create(<Select {...selectProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
