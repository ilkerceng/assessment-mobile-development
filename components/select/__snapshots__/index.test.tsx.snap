// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Select /> match snapshot 1`] = `
<RNGestureHandlerButton
  collapsable={false}
  onGestureEvent={[Function]}
  onGestureHandlerEvent={[Function]}
  onGestureHandlerStateChange={[Function]}
  onHandlerStateChange={[Function]}
  rippleColor={0}
  style={
    Array [
      Object {
        "overflow": "hidden",
      },
      undefined,
    ]
  }
>
  <View
    accessible={true}
    style={
      Object {
        "borderWidth": 1,
        "justifyContent": "center",
        "minHeight": 40,
        "opacity": 1,
        "width": "100%",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "row",
          "justifyContent": "space-between",
          "padding": 3,
        }
      }
    >
      <View
        style={
          Object {
            "flex": 1,
            "flexWrap": "nowrap",
          }
        }
      >
        <Text>
          Select...
        </Text>
      </View>
      <View>
        <Text />
      </View>
    </View>
  </View>
</RNGestureHandlerButton>
`;
